wagtail/wagtailimages/migrations/0004_make_focal_point_key_not_nullable.py

```diff
@@ -8,26 +8,34 @@ def remove_duplicate_renditions(apps, schema_editor):
     if schema_editor.connection.vendor == 'mysql':
         schema_editor.execute("""
             DELETE FROM `wagtailimages_rendition` WHERE CONCAT(image_id, '-', filter_id) IN (
-                SELECT CONCAT(image_id, '-', filter_id) FROM (SELECT * FROM `wagtailimages_rendition`) as x WHERE `focal_point_key` IS NULL GROUP BY image_id, filter_id HAVING COUNT(*) > 1
+                SELECT CONCAT(image_id, '-', filter_id)
+                FROM (SELECT * FROM `wagtailimages_rendition`) as x WHERE `focal_point_key` IS NULL GROUP BY image_id,
```

Comment:
Might be a little neater to break the line just before the WHERE and GROUP BY keywords